customization-base/src/main/java/com/azure/autorest/customization/implementation/ls/EclipseLanguageServerFacade.java

```diff
@@ -34,7 +34,7 @@ public EclipseLanguageServerFacade(String pathToLanguageServerPlugin, int port)
                 command += "-configuration ./config_linux";
             }
             server = Runtime.getRuntime().exec(command, new String[]{"CLIENT_PORT=" + port},
-                Paths.get(pathToLanguageServerPlugin, "jdt-language-server").toFile());
+                Paths.get(pathToLanguageServerPlugin, "../postprocessor/jdt-language-server").toFile());
```

Comment:
This probably won't work (or won't work alone) on released npm package.

I assume we need to modify the included `files` on npm package
https://github.com/Azure/autorest.java/blob/b1c855b0d641345ef4f3da5389b857aa81c1c1b1/typespec-extension/package.json#L41-L46

to include these as autorest npm package.
https://github.com/Azure/autorest.java/blob/b1c855b0d641345ef4f3da5389b857aa81c1c1b1/package.json#L40-L47

Reply:
Also, include the language server would likely double the package size :-)

Reply:
Discussed this offline and moved the language server to resources to include this with the jar. Also, the size of the package will be bigger similar to autorest but it's needed to make customization work. We'll have to investigate an alternative for Eclipse Language Server as a separate task to improve the efficiency of customization and package size.